Add the female developer joke (#65)
wesbos/dad-jokes: readme.md

**A:** Because he used up all his cache

---

> **Q:** What did the female developer that can not understand husband's behavior do?

Reviewer: ```suggestion
**Q:** What do you do when you can't understand your husband's behavior?
```

Author: @styfle 

Thank you to review!
